app/Helpers/database/user.php

```diff
@@ -282,7 +314,7 @@ function GetDeveloperStatsFull(int $count, int $sortBy, int $devFilter = 7): arr
                   WHERE $stateCond
                   GROUP BY ua.ID
                   ORDER BY $order
-                  LIMIT $count";
+                  LIMIT $offset, $count";
```

Comment:
Offset doesn't seem to be working when sorting by tickets resolved:
http://localhost:64000/developerstats.php?t=4&f=4&o=100

Or open tickets:
http://localhost:64000/developerstats.php?t=3&f=4&o=50

Or active claims:
http://localhost:64000/developerstats.php?t=7&f=1&o=50